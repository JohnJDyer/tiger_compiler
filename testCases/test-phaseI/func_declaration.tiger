let
  function a() begin flush(); end;
  function b(a: int) begin flush(); end;
  function c(a: int, b: float) begin flush(); end;
  function d(): int begin flush(); end;
in
  flush();
end