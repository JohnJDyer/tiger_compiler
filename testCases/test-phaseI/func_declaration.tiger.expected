a:
	call, flush
	return, , , 
b:
	call, flush
	return, , , 
c:
	call, flush
	return, , , 
d:
	call, flush
	return, , , 
main:
	call, flush
	return, , , 
